radicle-httpd/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Build
+FROM rust:1.65.0-slim@sha256:27349440d16ac15f52e8cae6181999a1b2fc05272d612a358d0e53078fcef88e as build
+
+RUN apt-get update && apt-get install -y pkg-config libssl-dev git cmake
+
+COPY . /app
+WORKDIR /app/radicle-httpd
+
+RUN set -eux; \
+    cargo install --locked --path .; \
+    objcopy --compress-debug-sections /usr/local/cargo/bin/radicle-httpd /usr/local/cargo/bin/radicle-httpd.compressed
```

Comment:
Can we use multiple `RUN` lines here instead of `\`?

Reply:
Sure thing